docs: more details for infoblox configuration (#172)
openshift/external-dns-operator: docs/infoblox-openshift.md

- Once the EC2 instance passed all the checks you can try to connect to the Grid Manager WebUI using the Elastic IP and `admin/${GRID_ADMIN_PASSWORD}` as credentials. Use HTTPS scheme and accept the self signed certificate

### Infoblox configuration
- Setup a new grid. At the first start a wizard would pop up and propose to do so, you can follow `Use vNIOS Instance for New grid` chapter from the guide in the [links](#links) or follow these notes:

>     - Most of the values can be kept as set by default however you may want to tweak the following options:

Reviewer: If it is optional to tweak the options, say what happens if you don't.  Otherwise, positively recommend to tweak them.
```suggestion
    - Most of the values can be kept as set by default except for these:
```
